tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -418,11 +421,12 @@ TangoErrorType TangoRosNode::TangoSetupConfig() {
 Tango3DR_Status TangoRosNode::TangoSetup3DRConfig() {
   const char* function_name = "TangoRosNode::TangoSetup3DRConfig()";
 
-  Tango3DR_ConfigH t3dr_config =
-      Tango3DR_Config_create(TANGO_3DR_CONFIG_CONTEXT);
+  Tango3DR_Config t3dr_config =
+      Tango3DR_Config_create(TANGO_3DR_CONFIG_RECONSTRUCTION);
   Tango3DR_Status result;
   const char* resolution = "resolution";
-  result = Tango3DR_Config_setDouble(t3dr_config, resolution, 0.05);
+  result = Tango3DR_Config_setDouble(t3dr_config, resolution,
+                                     tango_ros_conversions_helper::OCCUPANCY_GRID_RESOLUTION);
```

Comment:
Is this a value that a user might want to change depending on the scale of his application?